tools/docker/docker-compose-mac.yml

```diff
@@ -18,6 +18,7 @@ x-environment:
   - EDA_ANSIBLE_RULEBOOK_LOG_LEVEL=${EDA_ANSIBLE_RULEBOOK_LOG_LEVEL:-'-v'}
   - EDA_CONTAINER_NAME_PREFIX=${EDA_CONTAINER_NAME_PREFIX:-eda}
   - EDA_PODMAN_ENV_VARS=${EDA_PODMAN_ENV_VARS:-'@none None'}
+  - EDA_PGSSLMODE=${EDA_PGSSLMODE:-disable}
```

Comment:
@msmagnanijr if we have to enable this for our local dev enviornment what would we have to change?

Reply:
@mkanoor I realize that I should have included the variables in the 'compose' files and in Kubernetes deployments. I am wondering about the 'development.py' file.  Do you think this covers all the necessary aspects?

Reply:
If we merge [this](https://github.com/ansible/eda-server/pull/537), we would not need to add anything for minikube